src/texts/descriptions/aboutUs.tsx

```diff
@@ -27,6 +27,13 @@ export const visionDescription = (
   </>
 )
 
+export const logoDescription = (
```

Comment:
if it's pure text is it necessary to be an element? would a string of text work instead?

Reply:
true, string will do